lib/src/ruby/module.cc

```diff
@@ -297,12 +297,8 @@ namespace facter { namespace ruby {
         for (auto dir : paths) {
             _additional_search_paths.emplace_back(dir);
 
-            // Get the canonical directory name
-            boost::system::error_code ec;
-            path directory = canonical(_additional_search_paths.back(), ec);
-            if (ec) {
-                continue;
-            }
+            // Get the absolute directory name
```

Comment:
I just don't want someone else to come along later and not research why this change was made and go back to canonical.